packages/ui-design-system/src/utils.ts

```diff
@@ -0,0 +1,4 @@
+import { clsx, type ClassValue } from 'clsx';
+import { twMerge } from 'tailwind-merge';
+
+export const cn = (...classLists: ClassValue[]) => twMerge(clsx(classLists));
```

Comment:
I do think that TwMerge is necessary because we cannot update simple style without creating a new variant and sometimes it is not necessary, for example on the TestRun detail page, I cannot style to `text-grey-100` all the button we can toggle for the graph's legend